proto/penumbra/penumbra/core/transaction/v1/transaction.proto

```diff
@@ -54,6 +54,17 @@ message TransactionParameters {
   component.fee.v1.Fee fee = 3;
 }
 
+// Represents an individual effect of a transaction.
+message Effects {
+  keys.v1.AddressView address = 1;
+  asset.v1.Balance balance = 2;
+}
```

Comment:
Let's move this into a sub-message so it's clear that it's logically scoped to be part of the TransactionSummary.

Reply:
moved into a sub-message

```ts
// Represents a transaction summary containing multiple effects.
message TransactionSummary {
  // Represents an individual effect of a transaction.
  message Effects {
    keys.v1.AddressView address = 1;
    asset.v1.Balance balance = 2;
  }
  repeated Effects effects = 1;
}
```